content/en/docs/instrumentation/js/instrumentation.md

```diff
@@ -10,21 +10,21 @@ This guide will cover creating and annotating spans, creating and annotating met
 In the following this guide will use the following sample app:
 
 ```javascript
-'use strict';
+"use strict";
 
 for (let i = 0; i < 10; i += 1) {
   doWork();
 }
 
 function doWork() {
-  console.log("work...")
+  console.log("work...");
   // simulate some random work.
-  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {
-  }
+  for (let i = 0; i <= Math.floor(Math.random() * 40000000); i += 1) {}
```

Comment:
not related to this PR which fixes typos, but the line `Math.floor(Math.random() * 40000000)` will be re-evaluated again and again for each iteration which is probably not what was intended